bot/src/db.rs

```diff
@@ -86,6 +101,43 @@ pub struct EpochClassificationV1 {
 
     // Informational notes regarding this epoch
     pub notes: Vec<String>,
+
+    // Config values from Config struct
+    pub config: Option<EpochConfig>,
+
+    // General info about the Epoch
+    pub stats: Option<EpochStats>,
+}
+
+#[derive(Clone, Deserialize, Serialize)]
+pub struct EpochStats {
+    pub bonus_stake_amount: Option<u64>,
+    pub min_epoch_credits: u64,
+    pub avg_epoch_credits: u64,
+    pub max_skip_rate: usize,
+    pub cluster_average_skip_rate: usize,
+    pub total_active_stake: u64,
+}
+
+#[derive(Clone, Deserialize, Serialize)]
+pub struct EpochConfig {
+    pub require_classification: Option<bool>,
```

Comment:
I think we can drop all the Options from this struct, since the struct is being introduced here for the first time and there's not data files that serialize it.  Additional fields added to this struct in the future would need to be optional though

Reply:
Some of the config values really are optional, like `min_solana_version`, and it seemed at least possible that some of the other configuration values might get removed or become optional. Does that make sense? I can of course make the non-optional ones non-Optional if you think that's a good idea.

Reply:
Ah, yep that makes sense to me, thanks